web/config/settings.py

```diff
@@ -31,6 +31,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get('DEBUG') == 'True'
 
+DEBUG = True
```

Comment:
this broke the app. please revert

Reply:
This is required for development stage. It will be changed after developing.

Reply:
Please set `DEBUG` env var  to True, rather than hard-coding it.